src/EasyLiquidator.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.13;
+
+import "yield-utils-v2/token/IERC20.sol";
+import "yield-utils-v2/token/TransferHelper.sol";
+import "./interfaces/IERC3156FlashBorrower.sol";
+import "./interfaces/IERC3156FlashLender.sol";
+import "./interfaces/ISimpleSwap.sol";
+import "./interfaces/ICollateralizedVault.sol";
+
+
+contract EasyLiquidator is IERC3156FlashBorrower {
```

Comment:
Please add a natspec block explaining what is that this contract does. I actually don't remember.